protractor.conf.js

```diff
@@ -78,5 +78,10 @@ if (process.env.CI) {
   delete config.plugins;
 }
 
+// Add custom Chrome options for local environments
+if (!process.env.CI && process.env.CHROME_OPTIONS) {
```

Comment:
The test e2e works fine, I would like just to know what kind of values `CHROME_OPTIONS` can take, also if it is possible, add them in the comment as an example

Reply:
@mbayopanda I added a little more info on the PR description at the top. 
Thanks for testing it!